en/docs/setup/deployment-patterns.md

```diff
@@ -288,11 +288,123 @@ Following configurations need to be done to both WSO2 Identity Server nodes in o
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>3. No caching</td>
+                        <td>Disabled</td>
+                        <td>Disabled</td>
+                        <td>Disabled</td>
+                        <td>Disabled</td>
+                        <td>
+                            <ul>
+                                <li>The data are directly acquired from the database.</li>
+                                <li>Eliminates the security risks caused due to not having cache invalidation.</li>
+                                <li>This method will create a performance degradation due to the lack of caching.</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>4. Certain caches are disabled while the remaining are local</td>
+                        <td>Enabled for the available local caches</td>
+                        <td>Not Applicable</td>
+                        <td>Enabled</td>
+                        <td>Enabled</td>
+                        <td>
+                            <ul>
+                                <li>To reduce the security risk created in the second scenario and to improve performance in comparison with the third scenario, disable the security-related caches and sustain the performance-related caches as local caches.</li>
+                                <li>This requires identification of these caches depending on the use case.</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>5. Distributed caching enabled</td>
+                        <td>Disabled—the `force_local_cache` is set to `false`.</td>
+                        <td>Enabled</td>
+                        <td>Enabled</td>
+                        <td>Not Applicable</td>
+                        <td>
+                            <ul>
+                                <li>This scenario is only recommended if the network has tight tolerance where the network infrastructure is capable of handling high bandwidth with very low latency.</li>
+                                <li>Typically this applies only when you deploy <b>all the nodes in a single server rack having fiber-optic cables</b>. In any other environments, this implementation will cause cache losses. Thus, this implementation is <b>not recommended for general use</b>.
+                            </ul>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            </li>
+        </ul>
 
 3. Go to the `<IS_HOME>/repository/conf/deployment.toml` file and add the proxy port as `443`. The port 443 is the Load Balancer frontend port.
 
```

Comment:
to add the proxy port as `443`, the toml configuration is, 
[transport.https]
proxyPort = "443"

Reply:
Fixed